test_scripts/MobileProjection/Defects/3142_02_Presence_of_OnVideoDataStreaming_when_start_stop_streaming.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3140
```

Comment:
```suggestion
-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3142
```

Reply:
@GetmanetsIrina Done in 3e5e4043183b174da3ff9483cfa659905874a6dc

Reply:
@GetmanetsIrina Done in 3e5e404